.travis.yml

```diff
@@ -1,19 +1,14 @@
 os:
   - linux
-  - osx
-language: generic
+  #- osx
```

Comment:
I was just narrowing down the issue and wanted to make things complete faster. :-) I'll add it back in.